npm install error when updating to final-form v4.20.6

> npm v8.1.4

**package.json**
```
{
   ...
   "dependencies": {
      ...
      "final-form": "^4.20.6",
      "final-form-arrays": "^3.0.2",
      "final-form-submit-listener": "^1.0.0",
      "react-final-form": "^6.5.7",
      "react-final-form-arrays": "^3.1.3"
   }
}
```

**npm install error**
```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: www@0.0.0
npm WARN Found: final-form@4.20.4
npm WARN node_modules/final-form
npm WARN   final-form@"^4.20.6" from the root project
npm WARN   4 more (final-form-arrays, final-form-submit-listener, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer final-form@"4.20.4" from react-final-form@6.5.7
npm WARN node_modules/react-final-form
npm WARN   react-final-form@"^6.5.7" from the root project
npm WARN   1 more (react-final-form-arrays)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: www@0.0.0
npm WARN Found: final-form@4.20.4
npm WARN node_modules/final-form
npm WARN   final-form@"^4.20.6" from the root project
npm WARN   4 more (final-form-arrays, final-form-submit-listener, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer final-form@"4.20.4" from react-final-form@6.5.7
npm WARN node_modules/react-final-form
npm WARN   react-final-form@"^6.5.7" from the root project
npm WARN   1 more (react-final-form-arrays)
```

**It seems that you need to update the version of "final-form" in "peerDependencies" of "react-final-form".**

--- a/package.json
+++ b/package.json
@@ -84,7 +84,7 @@
     "typescript": "^4.4.3"
   },
   "peerDependencies": {
-    "final-form": "4.20.4",
+    "final-form": "^4.20.4",
     "react": "^16.8.0 || ^17.0.0"
   },
   "lint-staged": {

